src/renderer/components/Splash.js

```diff
@@ -70,6 +72,20 @@ class Splash extends React.Component {
   render() {
     let {classes, processing} = this.props;
     let disabled = processing ? 'disabled' : '';
+
+    const orDropSidebar = localize("splash.orDropSidebar");
+    const orDropSidebarArray = orDropSidebar.split("__");
+    const orDropSidebar1 = orDropSidebarArray[0].trim();
+    const orDropSidebar2 = orDropSidebarArray[1].trim();
+
+    // "browseForFileOrFolder": "browse for a __file__ or a __folder__ ."
+    const browseForFileOrFolder = localize("splash.browseForFileOrFolder");
+    const browseForFileOrFolderArray = browseForFileOrFolder.split("__");
+    const browseForFileOrFolder1 = browseForFileOrFolderArray[0].trim();
+    const browseForFileOrFolder2 = browseForFileOrFolderArray[1].trim();
+    const browseForFileOrFolder3 = browseForFileOrFolderArray[2].trim();
+    const browseForFileOrFolder4 = browseForFileOrFolderArray[3].trim();
+    const browseForFileOrFolder5 = browseForFileOrFolderArray[4].trim();
```

Comment:
This is ugly as hell, but this works fine for a first shot. TODO: regexp

Reply:
PS: `i18next` supports templates of course...but I haven't looked into breaking-down this particular splash message into a template-compatible form.